pallets/automation-time/src/tests.rs

```diff
@@ -78,102 +82,19 @@ fn schedule_past_time() {
 #[test]
 fn schedule_too_far_out() {
 	new_test_ext(START_BLOCK_TIME).execute_with(|| {
+		let call: RuntimeCall = frame_system::Call::remark { remark: vec![12] }.into();
 		assert_noop!(
-			AutomationTime::schedule_notify_task(
+			AutomationTime::schedule_dynamic_dispatch_task(
 				RuntimeOrigin::signed(AccountId32::new(ALICE)),
 				vec![50],
-				vec![SCHEDULED_TIME + 1 * 24 * 60 * 60],
-				vec![12]
+				ScheduleParam::Fixed { execution_times: vec![SCHEDULED_TIME + 1 * 24 * 60 * 60] },
+				Box::new(frame_system::Call::remark { remark: vec![12] }.into())
 			),
 			Error::<Test>::TimeTooFarOut,
 		);
 	})
 }
 
-#[test]
-fn schedule_no_message() {
-	new_test_ext(START_BLOCK_TIME).execute_with(|| {
-		assert_noop!(
-			AutomationTime::schedule_notify_task(
-				RuntimeOrigin::signed(AccountId32::new(ALICE)),
-				vec![50],
-				vec![SCHEDULED_TIME],
-				vec![]
-			),
-			Error::<Test>::EmptyMessage,
-		);
-	})
-}
-
-#[test]
-fn schedule_no_provided_id() {
-	new_test_ext(START_BLOCK_TIME).execute_with(|| {
-		assert_noop!(
-			AutomationTime::schedule_notify_task(
-				RuntimeOrigin::signed(AccountId32::new(ALICE)),
-				vec![],
-				vec![SCHEDULED_TIME],
-				vec![12]
-			),
-			Error::<Test>::EmptyProvidedId,
-		);
-	})
-}
-
-#[test]
-fn schedule_not_enough_for_fees() {
```

Comment:
I believe that `schedule_not_enough_for_fees` should be retained. Please use the `schedule_dynamic_dispatch_task` call to implement this test.

Reply:
Yes, I agree. schedule_not_enough_for_fees should be rewritten. Let’s discuss it Vinh